source/ui/BioXAS/BioXASSSRLMonochromatorConfigurationView.h

```diff
@@ -78,8 +67,22 @@ protected slots:
 	/// The crystal 2 roll editor.
 	AMExtendedControlEditor *crystal2RollEditor_;
 
+	/// The region editor.
+	BioXASSSRLMonochromatorRegionControlEditor *regionEditor_;
 	/// The region status display.
 	BioXASSSRLMonochromatorRegionControlView *regionStatusWidget_;
+
+	/// The step-based energy editor.
+	AMExtendedControlEditor *stepEnergyEditor_;
+	/// The encoder-based energy editor.
+	AMExtendedControlEditor *encoderEnergyEditor_;
+	/// The step-based bragg editor.
+	AMExtendedControlEditor *stepBraggEditor_;
+	/// The encoder-based bragg editor.
+	AMExtendedControlEditor *encoderBraggEditor_;
```

Comment:
The encoder editors were definitely to be removed from the PersistentView. It was requested that read-only editors for the encoder values be added to the ConfigurationView.